lib/primary_connect_client/configuration.rb

```diff
@@ -6,7 +6,7 @@
 The version of the OpenAPI document: v1
 
 Generated by: https://openapi-generator.tech
-OpenAPI Generator version: 5.3.0-SNAPSHOT
+OpenAPI Generator version: 5.2.1-SNAPSHOT
```

Comment:
Am I using an old version of OpenAPI Generator?

Reply:
Looks like the `latest` tag on the image is pointing to `5.2.1`, there is another release as of two months ago that is version `5.4.0` and its aliases to the tag `latest-release`.

We ought to lock our version of the generator in the `docker run` command as such:
from: `docker run --rm -v $PWD:/local openapitools/openapi-generator-cli generate --additional-properties=moduleName=PrimaryConnectClient --additional-properties=gemName=primary_connect_client -i /local/swagger/v1/swagger.yaml -g ruby -o /local/tmp/ruby`

to `docker run --rm -v $PWD:/local openapitools/openapi-generator-cli:v5.3.0 generate --additional-properties=moduleName=PrimaryConnectClient --additional-properties=gemName=primary_connect_client -i /local/swagger/v1/swagger.yaml -g ruby -o /local/tmp/ruby`